umbra-js/hardhat.config.ts

```diff
@@ -0,0 +1,102 @@
+import { config as dotenvConfig } from 'dotenv';
+import { resolve } from 'path';
+dotenvConfig({ path: resolve(__dirname, './.env') });
```

Comment:
Just curious—why do we need this setup instead of just `require('dotenv').config()`?

Reply:
copied verbatim from paulrberg template